### PR TITLE
Fixed the process of replacing certain symbols. (for #89 issue)

### LINKED ISSUE #89: No information is displayed when clicking the file name link in the "Check Imported CTI Elements" dialog.
If the attached file name contains characters that cannot be used in the URL (reserved character), no information is displayed when clicking the file name link in the "Check Imported CTI Elements" dialog.
Therefore, if the attached file name contains characters that cannot be used in the URL (reserved character), the process of replacing them is necessary.

---

添付するファイル名に URL で利用できない文字（予約文字）が含まれている場合、 "Check Imported CTI Elements" ダイアログでファイル名のリンクをクリックしても情報が表示されない。
そのため、添付ファイル名に URL で利用できない文字（予約文字）が含まれている場合は置換する処理が必要です。

### PR DESCRIPTION
**Pull request by ta-hiroki**

### CODE REVIEW

File: src/feeds/static/js/feeds.js [hunk 1]
```diff
@@ -269,7 +269,7 @@ $(function () {
   function confirm_indicators(){
     //modal dialog に Indicators の結果を table として追加する
     function get_file_id_from_file_name(file_name){
-    	return file_name.replace(/\./g,'--').replace(/ /g,'--').replace(/'/g,'--').replace(/"/g,'--').replace(/\(/g,'--').replace(/\)/g,'--').replace(/\:/g,'--').replace(/\//g,'--').replace(/\?/g,'--').replace(/=/g,'--')
+        return file_name.replace(/[\. '"\(\)\:\/\?=\[\]\{\}\#\@\!\$\&\+\,\;]/g,'--');
```
Comment (tkhsknsk) [inline src/feeds/static/js/feeds.js]: Please adjust it to 2 indents.

Comment (ta-hiroki) [reply src/feeds/static/js/feeds.js]: The correction was carried out.

**Review by tkhsknsk — APPROVED** [review]

**Review by stmtstk — CHANGES_REQUESTED** [review]
When an user posts with two files whose name is below...

```
test(1).txt
test{1}.txt
```

After posting, the user clicks `test{1}.txt` in CTI Element Dialog Box, `test(1).txt` is opened.

Please check the behavior.

**Comment by ta-hiroki** [comment]
> https://github.com/s-tip/stip-sns/pull/92#pullrequestreview-503569897

@stmtstk 
I confirmed the content of the point.
Replace the "--" with the "--ASCII Code".

**Comment by stmtstk** [comment]
Thank you. I confirmed it.
However, I am not sure that we should modify `confirm_indicators` in `search.js` as well.

@tkhsknsk , could you tell me whether we should modify it or not?

---

修正は確認できました。
しかし、search.js にも `confirm_indicators` があります。
search.js の中に feeds.js の一部を移植した修正を @tkhsknsk さんが行っていたと思うのですが、
`search.js` にも必要であるか教えてください。
必要であれば @ta-hiroki  さん、対応お願いします。

**Comment by ta-hiroki** [comment]
@stmtstk 

Thank you for your confirmation.
"seach.js" has a similar occurrence.
I will fix it in the same way.

---

確認ありがとうございます。
seach.js にも同様の機能があります。
同じように修正を実施いたします。

**Review by stmtstk — APPROVED** [review]

**Comment by stmtstk** [comment]
Thanks for your contribution.

**Pull request merged**